SUMMARY.md

```diff
@@ -12,6 +12,8 @@
   * [Reputation management](use-cases/reputation-management.md)
   * [Revenue management](use-cases/revenue-management.md)
   * [Upselling](use-cases/upselling.md)
+  * Payment automation
+     * [Using Mews' integrated payment gateway for credit card payments](/use-cases/payment-automation/tokenized-credit-cards.md)
```

Comment:
Not sure if this formatting will show up correctly on Gitbook. Please suggest better title (if any)